electrum/gui/kivy/main_window.py

```diff
@@ -1200,6 +1200,21 @@ def on_success(old_password, new_password):
                                    on_success=on_success, on_failure=on_failure, is_change=1)
         self._password_dialog.open()
 
+    def save_backup(self):
+        from .uix.dialogs.password_dialog import PasswordDialog
+        from electrum.util import get_backup_dir
+        if self._password_dialog is None:
+            self._password_dialog = PasswordDialog()
+        message = _("Create backup.") + '\n' + _("Enter your current PIN:")
+        def on_success(old_password, new_password):
+            new_path = os.path.join(get_backup_dir(self.electrum_config), self.wallet.basename() + '.backup')
```

Comment:
Maybe we could put a timestamp in the name?
To avoid backups overwriting each other. Or is it a feature that the new one overwrites the old one?

Reply:
better overwrite it. in fact, I think we should automate this, as in setup a backup path only once, and save it on each new channel creation.